units/XNA0104/XNA0104_Script.lua

```diff
@@ -204,9 +208,9 @@ XNA0104 = Class(NAirTransportUnit) {
     end,
 
     ---@param self XNA0104
-    ---@return number
-    ---@return number
-    ---@return number
+    ---@return number blueprints SizeX
+    ---@return number blueprints SizeY 
+    ---@return number blueprints SizeZ
```

Comment:
Looking at the code, it is not always the blueprint `SizeX` value that is returned.
```suggestion
    ---@return number X size
    ---@return number Y size 
    ---@return number Z size
```